src/MassTransit/Monitoring/StartedInstrument.cs

```diff
@@ -0,0 +1,29 @@
+#nullable enable
+// ReSharper disable once CheckNamespace
+namespace MassTransit.Logging
```

Comment:
Yeah, seriously, just move all this stuff to the Logging folder, or a Monitoring folder _under_ Logging with a namespace provider = false.